engine/src/main/java/org/hibernate/search/analyzer/impl/LuceneEmbeddedAnalyzerStrategy.java

```diff
@@ -35,7 +35,7 @@
  */
 public class LuceneEmbeddedAnalyzerStrategy implements AnalyzerStrategy {
 
-	private static final Log log = LoggerFactory.make();
+	private static final Log LOG = LoggerFactory.make();
```

Comment:
I'm ok to change this, but TBH I always thought it was weird that you would create `LOG` instances... traditionally we've always just called them `Log`, as it's not really a constant (it's a mutable object).

No need to change anything..